### PR TITLE
Add download to export command

### LINKED ISSUE #139: Update export to download csv to home directory
Currently the export command only downloads the .csv file to the data directory within the project for further use. However, the user is not currently able to access the csv file unless they have the full project repository cloned on the device.

Let's update the export command to download a copy of the .csv file to the user's home directory as well. Additional checks should be added as well to check for overwriting in both the data directory and the user's home directory.

### PR DESCRIPTION
**Pull request by yhanyi**
Fixes #139 

The export command currently only creates a .csv in the data directory of the project folder. This update allows the export command to copy a .csv file into the user's home directory and updates tests.

### CODE REVIEW

**Comment by Clarenceeey** [comment]
LGTM

File: src/main/java/seedu/address/logic/commands/ExportCommand.java [hunk 1]
```diff
@@ -63,24 +68,84 @@ public CommandType getCommandType() {
...
+                // Ignore cleanup errors
+            }
+            throw new CommandException(String.format(MESSAGE_FAILURE, e.getMessage()));
+        }
+    }
 
-        // Check if file exists and force flag is not set
-        if (!isForceExport && Files.exists(filePath)) {
-            throw new CommandException(String.format(MESSAGE_FILE_EXISTS, filePath));
+    String escapeSpecialCharacters(String data) {
```
Comment (notnotmax) [inline src/main/java/seedu/address/logic/commands/ExportCommand.java:147]: Out of curiosity, what does this do? Why do we need to escape special characters?

**Review by notnotmax — COMMENTED** [review]
lgtm but I'm not very knowledgeable on file operations, so might want to wait for a second reviewer to approve this as well

File: src/main/java/seedu/address/logic/commands/ExportCommand.java (hunk 1 above)
Comment (yhanyi) [reply src/main/java/seedu/address/logic/commands/ExportCommand.java:147]: It's more of a utility function to help with .csv formatting throughout the file.

**Comment by marcusjhang** [comment]
lgtm

**Pull request merged**